Makefile

```diff
@@ -1,20 +1,19 @@
 .PHONY: all binary ut clean
 
 SRCDIR=calico_kubernetes
-BUILD_DIR=build_calico_kubernetes
+BUILD_DIR=.
 BUILD_FILES=$(BUILD_DIR)/Dockerfile $(BUILD_DIR)/requirements.txt
 
-
 default: all
 all: binary test
 test: ut
 
 # Build a new docker image to be used by binary or tests
-kubernetesbuild.created: $(BUILD_FILES)
-	cd build_calico_kubernetes; docker build -t calico/kubernetes-build .
-	touch kubernetesbuild.created
+kubernetescnibuild.created: $(BUILD_FILES)
```

Comment:
I'd call this "buildcontainer.created" -- That is actually what it is, a build container.